MAUI/PDF-Viewer/Free-Text.md

```diff
@@ -217,4 +229,4 @@ Private void customDialogOkButton_Clicked(object sender, EventArgs e)
 {% endhighlight %} 
 {% endtabs %}
 
-N> For WinUI and MacCatalyst platforms, there is no separate modal view to receive text input from the users. As a result, the `FreeTextModalViewAppearing` and `FreeTextModalViewDisappearing` events are not applicable for these platforms. 
\ No newline at end of file
+N> For WinUI and MacCatalyst platforms, there is no separate modal view to receive text input from the users. As a result, the `FreeTextModalViewAppearing` and `FreeTextModalViewDisappearing` events are not applicable for these platforms. 
```

Comment:
There is unwanted black border appears. can you please remove this?

<img width="450" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/f911b499-a4fb-4561-83c2-47727d501e0d">

Reply:
Feedback addressed